README.md

```diff
@@ -1,26 +1,25 @@
 - [Overview](#Overview)
-  - [Purpose & Principles](#Purpose-&-Principles)
+  - [Purpose](#Purpose)
+  - [Principles](#Principles)
   - [Architecture Diagram](#Architecture-Diagram)
 - [Features](#Features)
 - [QuickStart](#QuickStart)
   - [Download](#Download)
   - [Build From Source](#Build-From-Source)
-- [Documentation](#Documentation)
-  - [Agent Configuration](#Agent-Configuration)
-  - [Health Check Endpoint](#Health-Check-Endpoint)
-  - [Readiness Check Endpoint](#Readiness-Check-Endpoint)
-  - [Tracing](#Tracing)
-  - [Metrics](#Metrics)
-    - [Prometheus Metrics Support](#Prometheus-Metrics-Support)
+- [User Manual](#User-Manual)
 - [Development Guide](#Development-Guide)  
 - [Licenses](#Licenses)
 
 
 ## Overview
+An agent component for Java system
 
 ### Purpose
-EaseAgent is an APM tool under the Java system, used in a distributed system developed by Java.
-It provides cross-service call chain tracking and performance information collection for distributed systems, helps users analyze the overall structure of the distributed system and the call relationships between services and components, thereby improving the efficiency of troubleshooting.
+- Lightweight
+- Low resource usage
```

Comment:
Lightweight and Low resource usage shouldn't be placed under the `Purpose` section.

Reply:
I think there are wrongs in our meeting minutes.  Can @haoel confirm this?

Reply:
Yes, Lightweight and Low resource belong to Principles.
I will fix it